packages/stardog-graphql-language-server/src/GraphQlLanguageServer.ts

```diff
@@ -188,21 +188,31 @@ export class GraphQlLanguageServer extends AbstractLanguageServer<
         update.relationshipBindings || this.relationshipBindings;
       this.relationshipCompletionItems = this.buildCompletionItemsFromData(
         this.namespaceMap,
-        this.relationshipBindings.map((binding) => ({
-          iri: binding.relationship.value,
-          count: binding.count.value,
-        })),
+        this.relationshipBindings
+          .map((binding) => ({
+            iri: binding && binding.relationship && binding.relationship.value,
+            count:
+              binding && binding.count && binding.count.value !== undefined
+                ? binding.count.value
+                : undefined,
```

Comment:
i think you could have done this since `({} && {} && undefined) === undefined`, but these logic rules always mess me up so i don't mind either way

```suggestion
            count: binding && binding.count && binding.count.value,
```